roles/cluster_infra/tasks/main.yml

```diff
@@ -45,6 +45,35 @@
     - providers.tf
     - resources.tf
 
+- name: Find terraform outputs, just to initi tofu state
```

Comment:
We probably should extract all of this into something we can use more easily for other appliances.